Set port via cmd line argument and update README

ndoschek opened the pull request:
- Enable custom model server port via command line argument
- Update README.md

ndoschek committed:
Set port via cmd line argument and update README

tortmayr commented on com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerLauncher.java:
```diff
@@ -33,6 +33,8 @@
 	private String workspaceRoot;
 	private String[] args;
 
+	private static final int DEFAULT_JAVALIN_PORT = 8081;
```
Should be a public constant

tortmayr replied:
Should be a public constant

tortmayr commented on com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerLauncher.java:
```diff
@@ -57,7 +59,9 @@ public void start() {
 
 	protected void run() {
-		injector.getInstance(ModelServerStartup.class).boot(EntryPointType.REST, this.args);
+		int port = args.length > 0 ? Integer.parseInt(args[0].split("=")[1]) : DEFAULT_JAVALIN_PORT;
```
This causes an uncatched exceptions (`ArrayIndexOutOfBoundsException`, `NumberFormatException`) if the argument doesn't have the right format. (no `=` or the right-hand side of `=` cannot be cast to an integer)

We should catch this and print some information about invalid arguments and the correct usage.

In addition, the --port prefix is currently not checked. IMHO it should either be checked if the argument starts with "--port" or the prefix should be omitted.

ndoschek replied:
Thanks for testing, that's a good point!

tortmayr reviewed (changes requested):
Thanks for improving the readme!
I noticed a minor issue with the port parameter parsing. Other than that LGTM.

edgarmueller commented on README.md:
```diff
@@ -1,13 +1,35 @@
+
+## Run the modelserver
+### Run the modelserver in IDE
+To run the example modelserver within an IDE, run the main method of [ExampleServerLauncher.java](https://github.com/eclipsesource/modelserver/blob/master/examples/com.eclipsesource.modelserver.example/src/main/java/com/eclipsesource/modelserver/example/ExampleServerLauncher.java) as Java Application, located in the module `com.eclipsesource.modelserver.example`.
```
as **a** Java Application

edgarmueller commented on README.md:
```diff
@@ -1,13 +1,35 @@
+
+
+## Run the modelserver
+### Run the modelserver in IDE
```
**an** IDE

edgarmueller commented on README.md:
```diff
@@ -1,13 +1,35 @@
+To change the port, use the command line argument `--port=XXXX`.
+
+## Use the modelserver API
+If the modelserver is up and running, you can access the modelserver API via http://localhost:8081/api/model/:modeluri.
```
Use backticks for URL formatting

ndoschek committed:
Check arguments and update README
- Check if argument is valid and supported
- Print information otherwise
- Fix mistakes in README

tortmayr commented:
Follow-up Issue: #9
